ansible/playbooks/openshift/uninstall.yml

```diff
@@ -13,6 +13,13 @@
       shell: oc delete clusterroles -l app=enmasse
     - name: Delete API services
       shell: oc delete apiservices -l app=enmasse
+    - name: Check for console API
+      shell: oc api-resources --namespaced=false --api-group=console.openshift.io --no-headers
+      register: console_api
+      ignore_errors: True
+    - name: Delete consolelinks
+      shell: oc delete consolelinks -l app=enmasse
+      when: console_api.stdout.find('ConsoleLink') != -1
```

Comment:
Would it not be sufficient with only the 'Delete consolelinks' step?

Reply:
My intent was to separate the failure that would result on 3.11 as the resource is unknown from a failure that could occur on OCP 4 if something really went wrong.   Maybe I'm over complicating things?

Reply:
Ah, understood. I'm fine leaving this in.